examples/mvc/utils.py

```diff
@@ -0,0 +1,113 @@
+from typing import Dict, List, Tuple, Union, Any, Optional, Generator, Iterator, Hashable
+
+def binary2spin(linear: Dict[int, float],
```

Comment:
@ianmalcolm following our discussion today, let me suggest to remove `binary2spin` and keep just a single function `graph2hamiltonian(graph, penalty)`.